### ISSUE TITLE
Language Selection Post/Comment

### ISSUE THREAD

**Issue by Tealk** [Xyphyn/photon#182]
**Is your feature request related to a problem? Please describe.**
There are some communities that allow multiple languages, so it would be an advantage to be able to choose which language the content is in.

**Describe the solution you'd like**
A selection field for the language, there would still be enough space in the bar at the top.
![image](https://github.com/Xyphyn/photon/assets/12276250/d32650b4-88d0-4b40-9a8d-74a99aec8ec3)

**Additional context**
The language of the article is important for filtering out content that you don't want to/can't read and for translation tools that are based on it.

**Comment by Xyphyn** [comment]
Adding soon. Also, it seems you're using dark reader, which breaks the theming on photon.

**Comment by Tealk** [comment]
Yes, it is possible that I had not yet deactivated it in the screenshot, but it is now deactivated.

**Comment by Neshura87** [comment]
@Xyphyn are you working on this or is it still up for grabs?

**Comment by Xyphyn** [comment]
I will work for this soon
